DataFormats/Luminosity/interface/PixelClusterCountsPerEvent.h

```diff
@@ -0,0 +1,46 @@
+#ifndef DataFormats_PixelClusterCountsPerEvent_h
+#define DataFormats_PixelClusterCountsPerEvent_h
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h